projects/components/src/table/cells/data-parsers/table-cell-row-parser.ts

```diff
@@ -0,0 +1,21 @@
+import { TableRow } from '../../table-api';
+import { TableCellParser } from '../table-cell-parser';
+import { TableCellParserBase } from '../table-cell-parser-base';
+import { CoreTableCellParserType } from '../types/core-table-cell-parser-type';
+
+@TableCellParser({
+  type: CoreTableCellParserType.Row
+})
+export class TableCellRowParser extends TableCellParserBase<unknown, TableRow, unknown> {
+  public parseValue(_: unknown, rowData: TableRow): TableRow {
```

Comment:
Given that the renderers have access to both the row and cell, do we need a row parser? Would `TableCellNoOpParser` serve this use case?

Reply:
But it wouldn't assign rowData to value. I don't need to use value property, but this is how it is designed

Reply:
Lemme know. I wouldn't it be weird for the design? I can use fields from constructor, but i hope it doesn't break any other use case

Reply:
It shouldn't. All the base class does is set some values, but the renderer ultimately can use whatever it wants. If we have a renderer that's doing something at the row level, it's likely custom anyway